### ISSUE TITLE
documentation error for CM4 - no such file or directory: '/tmp/klipper_host_mcu

### ISSUE THREAD

**Issue by sredoje1973** [bigtreetech/Pad7#10]
Hi there,
I like the Pad 7, got it working great with the supplied CB1. I decided to upgrade to CM4 and followed the instructions, and they do not work.  I added the following to the printer.cfg per the PDF

```
[mcu CM4]
serial: /tmp/klipper_host_mcu
[adxl345]
cs_pin: CM4:None
spi_bus: spidev0.1
axes_map: z,y,-x #
```

When doing to, I kept getting errors like 'MCU CM4' not connected. 

Looking at klippy.log I found the following error
**no such file or directory: '/tmp/klipper_host_mcu'**

Looking at the machine from a terminal, in fact there was no file there.    
After a ton of frustration,  I learned/realized that the host CM4 was not configured as a MCU for klipper!

In order to use the CM4, so that you can use the SPI interface in order to access the accelerometer a few more steps required.

**TLDR**:  follow the instructions here to build the firmware for the CM4 so that klipper can use the Linux machine as a processor, access it's I/O ports, etc.  **https://www.klipper3d.org/RPi_microcontroller.html**

In particular, follow these instructions

```

> Install the rc script[¶](https://www.klipper3d.org/RPi_microcontroller.html#install-the-rc-script)
> If you want to use the host as a secondary MCU the klipper_mcu process must run before the klippy process.
> 
> After installing Klipper, install the script. run:
> 
> 
> cd ~/klipper/
> sudo cp ./scripts/klipper-mcu.service /etc/systemd/system/
> sudo systemctl enable klipper-mcu.service
> Building the micro-controller code[¶](https://www.klipper3d.org/RPi_microcontroller.html#building-the-micro-controller-code)
> To compile the Klipper micro-controller code, start by configuring it for the "Linux process":
> 
> 
> cd ~/klipper/
> make menuconfig
> In the menu, set "Microcontroller Architecture" to "Linux process," then save and exit.
> 
> To build and install the new micro-controller code, run:
> 
> 
> sudo service klipper stop
> make flash
> sudo service klipper start
> 



then reboot
